website/blog/2023-05-02-release-0.15.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Release Notes - Podman Desktop 0.15
+description: Podman Desktop 0.15 has been released! 
+slug: podman-desktop-release-0.15
+authors: [deboer]
+tags: [podman-desktop, release, kubernetes, kind]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 0.15 - Cleanliness is next to Podliness!
+
+It has only been two weeks since our last release, but we really wanted to complete a few scenarios,
+fix a few bugs, and show off several design updates and UI improvements that we have been working on.
+
+<!--Main Features-->
+
+- **Podman Version**: Podman 4.5 now included in Windows and Mac installers.
+- **Kind Ingress**: Configure an ingress controller.
```

Comment:
Maybe "Creating an ingress?" We dont configure an ingress controller, only install.

Reply:
Fixed in latest commit.